module/VuFind/src/VuFind/Controller/AbstractSearch.php

```diff
@@ -352,6 +352,8 @@ protected function getRssSearchResponse(ViewModel $view): Response
     protected function getSearchResultsView($setupCallback = null)
     {
         $view = $this->createViewModel();
+        $config = $this->getConfig('facets');
+        $view->multiFacetsSelection = $config?->get('Advanced_Settings')?->get('multiFacetsSelection') === '1';
```

Comment:
For consistency with typical VuFind style, I'd change this to `$view->multiFacetsSelection = (bool)($config->Advanced_Settings->multiFacetsSelection ?? false);`